implementations/python/provekit-lift-py-tests/src/provekit_lift_py_tests/assertion_layer.py

```
@@ -0,0 +1,386 @@
+        if extra:
+            raise ValueError(
+                f"{name}: keyword arg(s) {sorted(extra)} may change the relation; "
+                "not liftable as exact equality"
+            )
+        for rk in vocab.require_true_kwargs:
+            if not _kwarg_is_true(call, rk):
+                raise ValueError(
+                    f"`{name}` is approximate by default; refused as exact equality "
+                    f"unless {rk}=True is pinned"
+                )
+        l = _translate_term_scoped(call.args[0], scope, call_vars)
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)